app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
```

Comment:
- [ ] Dear @Zven94, For the `users_controller` and `posts_controller` we need to have a `#index` and `#show` actions instead of the `new`. It's okay if these methods are empty but we need to create those actions.